scripts/install/sonarr.sh

```diff
@@ -19,7 +19,39 @@
 #
 #################################################################################
 
+function _check_for_sonarr3 () {
+  if [[ -f /install/.sonarrv3.lock ]]; then 
+    v3present=true
+  fi
+
+  if dpkg -l | grep sonarr > /dev/null 2>&1 ; then 
+    v3present=true
+  fi
+
+  if [[ $v3present == "true" ]]; then
+    echo "Sonarr v3 detected. If you want to proceed installing Sonarr v2, please remove Sonarr v3 first."
+    exit 1
+  fi
+
+  if [[ -d /root/sonarrv2.bak ]]; then 
+    echo
+    echo "WARNING: Found backups before Sonarr v3 installation"
+    echo "Follow these steps post-install https://github.com/Sonarr/Sonarr/wiki/Backup-and-Restore"
+    echo "The backup is stored in /root/sonarrv2.bak/"
```

Comment:
Can we put the backup somewhere other than just dumping it in the root dir? I find this folder can easily get cluttered.

I know it's not the best, but currently the deluge backup is stored right next to the deluge directory in .config. Otherwise, a standardized backup location?

Reply:
Sure thing, where should it go?

Reply:
`~/.config` is already being used, otherwise, `/root/backups` or if we intend to further extend in the future `/root/swizzin/backups`

Reply:
I like `/root/swizzin/` and `backups` in there, I'll go with that.